libs/utils/utils.js

```diff
@@ -1167,11 +1168,11 @@ export async function decoratePlaceholders(area, config) {
   const nodes = findReplaceableNodes(area);
   if (!nodes.length) return;
   area.dataset.hasPlaceholders = 'true';
-  const phPaths = getPlaceholderPaths(config);
+  const { decoratePlaceholderArea, getPlaceholderRoot } = await import('../features/placeholders.js');
```

Comment:
If there are replaceable nodes then we have to load placeholders.js.  I think we should move `getPlaceholderPaths` and the rest of this `decoratePlaceholders` function into placeholders.js in order to reduce code in utils.js.

Reply:
I agree, but I think that's a little out of scope for this ticket.
Migrating decoratePlaceholders will require some investigation and thorough testing for various use cases.